src/modules/70-pipeline/pages/execution/ExecutionArtifactsView/ArtifactsTable/ArtifactTableCells.tsx

```diff
@@ -174,17 +174,30 @@ export const TypeCell: CellType = ({ row }) => {
 }
 
 export const SLSAVerificationCell: CellType = ({ row }) => {
-  const { getString } = useStrings()
   const data = row.original
 
-  const verifyAttestationStatus = (): string => {
-    return `${getString('pipeline.attestationVerification')}: ${
-      data.provenance ? getString('passed') : getString('failed')
-    }`
-  }
+  const slsaPolicyOutcomeStatus = get(data.node, `outcomes.policyOutput.status`, undefined)
+
+  return (
+    <SLSAVerification
+      hasSlsaVerification={data.node?.stepType === StepType.SlsaVerification}
+      slsaPolicyOutcomeStatus={slsaPolicyOutcomeStatus}
+      provenance={data.provenance}
+    />
+  )
+}
+
+// common component shared to MFE
+export const SLSAVerification: FC<{
+  hasSlsaVerification: boolean
+  slsaPolicyOutcomeStatus: string | undefined
+  provenance: string | undefined
```

Comment:
Not a problem, but better using the optional syntax rather than saying `undefined`

```suggestion
  slsaPolicyOutcomeStatus?: string
  provenance?: string
```